contracts/RenPool.sol

```diff
@@ -201,7 +201,12 @@ contract RenPool {
         // TODO emit event
     }
 
-    // TODO: cancelWithdrawRequest
+    function cancelWithdrawRequest() external {
+        address sender = msg.sender;
+        require(withdrawRequests[sender] >= 0, "No withdraw request");
+
```

Comment:
I'm hesitating if we need the first 2 lines or not. Also, It'd be nice to add tests for this piece of code as well.

Reply:
I'll add the tests, as for the lines sender is used two times in the function so making it a variable makes sence in my opinion. 
I could also use msg.sender too of course

Reply:
I mean, not sure if we need the require statement here.